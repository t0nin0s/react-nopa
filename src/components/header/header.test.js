import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Header from './header'

test('Header component renders as expected in homepage', () => {
  const component = shallow(<Header home='true' />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})

test('Header component renders as expected', () => {
  const component = shallow(<Header />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
