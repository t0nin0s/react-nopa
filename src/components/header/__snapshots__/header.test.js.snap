// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders as expected 1`] = `
<header
  className="header"
>
  <Link
    replace={false}
    to="/"
  >
    <img
      className="logo"
      src={Object {}}
    />
  </Link>
  <div
    className="logInButton"
  >
    <div
      className="vCenterItems"
    >
      <Link
        className="button buttonHeader"
        replace={false}
        to="/login"
      >
        Log in
      </Link>
    </div>
  </div>
</header>
`;

exports[`Header component renders as expected in homepage 1`] = `
<header
  className="header"
>
  <Link
    replace={false}
    to="/"
  >
    <img
      className="logo logoHome"
      src={Object {}}
    />
  </Link>
  <div
    className="logInButton"
  >
    <div
      className="vCenterItems"
    >
      <Link
        className="button buttonHeader"
        replace={false}
        to="/login"
      >
        Log in
      </Link>
    </div>
  </div>
</header>
`;
